file.go

```diff
@@ -0,0 +1,42 @@
+package python
+
+/*
+#include <stdio.h>
+#include "go-python.h"
+
+PyObject*
+_gopy_PyFile_FromFile(int fd, char *name, char *mode) {
+    FILE *f = fdopen(fd, mode);
+    PyObject *py = PyFile_FromFile(f, name, mode, NULL);
+    PyFile_SetBufSize(py, 0);
+    return py;
+}
+
+*/
+import "C"
+
+import (
+	"os"
+)
+
+// FromFile converts a Go file to Python file object.
+// Calling close from Python will not close a file descriptor.
+func FromFile(f *os.File, mode string) *PyObject {
+	p := C._gopy_PyFile_FromFile(C.int(f.Fd()), C.CString(f.Name()), C.CString(mode))
```

Comment:
here, we are leaking the 2 strings.

please create 2 temporary `*C.char` variables that are `C.free`-ed later on.

Reply:
I was not sure if "name" pointer will not be used for PyFile object. If not, I will free it.